### ISSUE TITLE
Change base path of web server?

### ISSUE THREAD

**Issue by mateogianolio** [jhuckaby/Cronicle#680]
## Summary

Maybe this has been asked before but I have searched all the issues here and pixl-server-web and nothing really pops up. I want to host the cronicle web server under a reverse proxy at the subfolder /cronicle. Is there no way to set base path on the cronicle web server?

Setting `base_app_url` has no effect.

## Steps to reproduce the problem

Set up a reverse proxy with nginx that uses `proxy_pass`:

```nginx
location /cronicle {
    proxy_pass http://localhost:3012/;
    proxy_http_version 1.1;
    proxy_redirect off;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header Host $http_host;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection "upgrade";
    proxy_read_timeout 86400;
}
```

The html files load but the static assets (js/css etc) do not. This is expected since the base path in cronicle is `/`. I would like to be able to set base path to /cronicle.

## Your Setup

N/A

### Operating system and version?

N/A

### Node.js version?

N/A

### Cronicle software version?

N/A

### Are you using a multi-server setup, or just a single server?

Single serfver

### Are you using the filesystem as back-end storage, or S3/Couchbase?

N/A

### Can you reproduce the crash consistently?

N/A

## Log Excerpts

N/A

**Comment by jhuckaby** [comment]
I'm so sorry, but I don't know of any way to accomplish this.  Cronicle expects to be hosted on the root `/` URI path.

**Comment by mateogianolio** [comment]
Ok, thanks for the quick reply! Feel free to close this :)

**Comment by mikeTWC1984** [comment]
@mateogianolio  you can configure nginx to have your cronicle under http://somehost/cronicle. I think you need to provide some extra paths for socket.io.. Probably start in with browser dev tools / network, see what js files are missing.  Search for nginx in issues, I think this was discussed few times

**Comment by DmitryAdam** [comment]
Anyway, have you ever wondering how to backup and restore cronicle schedule from one server to another server?

**Comment by odahcam** [comment]
> I'm so sorry, but I don't know of any way to accomplish this. Cronicle expects to be hosted on the root `/` URI path.

This is crazy, would you accept a PR to add such functionality?

**Comment by jhuckaby** [comment]
No, sorry, Cronicle is in maintenance mode.  This would be too large and complex of a change.  I don't have the time to test and maintain such a thing.  Apologies.

Can you do a subdomain instead?  cronicle.yourdomain.com ?
